debian/changelog

```diff
@@ -1,3 +1,53 @@
+mir (2.10.0) UNRELEASED; urgency=medium
+
+  * New upstream release 2.10.0
+
+    - ABI summary:
+      . miral ABI unchanged at 5
+      . mircommon ABI unchanged at 9
+      . mircookie ABI unchanged at 2
+      . mircore ABI unchanged at 2
+      . miroil ABI unchanged at 2
+      . mirplatform ABI unchanged at 23
```

Comment:
I don't think the mirplatform changes (`EventBuilder` gained a virtual function; `WriteMappableBuffer` gained a virtual base class; and, `BufferRenderTarget::set_buffer()` changed signature) are ABI safe.